partiql-ast/src/main/resources/partiql_ast.ion

```diff
@@ -500,7 +500,7 @@ expr::[
     where:    optional::expr,
     group_by: optional::group_by,
     having:   optional::expr,
-    set_op:   optional::{
+    set_op:   optional::{ // TODO modeling of `set_op` needs updated to support left-associative set ops https://github.com/partiql/partiql-lang-kotlin/issues/1507
```

Comment:
The current modeling of SQL set ops is not correct in the AST (see https://github.com/partiql/partiql-lang-kotlin/issues/1507). I chose to just reuse the `Expr.BagOp` node rather than use this `Expr.SFW.SetOp` node for this PR to preserve backwards compatibility for this release.

We should fix the modeling before the `v1` release.